build/docs.js

```diff
@@ -220,6 +220,7 @@ module.exports = (gulp) => {
     gulp.task('java-mvn-gen', ['java-clone'], (callback) => {
         childProcess.exec(
             `cd ${config.src.tmp}/beam-client-java && mvn clean javadoc:javadoc`,
+            { maxBuffer: 1024 * 1024 * 10 },
```

Comment:
Not really, our docs just got bigger so the default (200 KB buffer) is sometimes exceeded which can result in errors being thrown. We could and maybe should do some stuff in the future to tell Node to discard standard output. A quick fix would be piping to /dev/null but that won't work on windows, the actual fix would be using `.spawn` instead of exec.